docs/process.md

```diff
@@ -174,6 +174,53 @@ updating `emcc.rst` in a PR, the following should be done:
 See notes above on installing sphinx.
 
 
+## Updating the LLVM libraries
+
+We maintain our ports of compiler-rt, libcxx, libcxxabi, and libunwind under
+https://github.com/emscripten-core/emscripten/tree/main/system/lib from [the
+upstream LLVM repository][llvm_repo] and periodically update them to a newer
```

Comment:
I don't think `[ ]` links work if split over two lines when rendered? Maybe shorten the first line here slightly,
```suggestion
https://github.com/emscripten-core/emscripten/tree/main/system/lib from
[the upstream LLVM repository][llvm_repo] and periodically update them to a newer
```

Reply:
The link seems to work, but yeah it might be better to gather `[ ]` in a single line.